docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -52,9 +52,9 @@ Once the above steps are complete, you'll be able to log in (using your IDIR) to
 
 ## Background: BC government GitHub Enterprise process map
 
-Our team has created the artifact below which captures the steps and teams involved in acquiring and mananaging GitHub Enterprise licenses. This version is intended for reference only, but we intend to create simplified, more focused versions for both end users and administrators.  
+Our team has created the artifact below which captures the steps and teams involved in acquiring and mananaging GitHub Enterprise licenses. This version is intended for reference only.
```

Comment:
```suggestion
The [Developer Experience Team](developer.experience@gov.bc.ca) has created the artifact below which captures the steps and teams involved in acquiring and mananaging GitHub Enterprise licenses. This version is intended for reference only.
```

Reply:
Maybe put in our team name, so people who skim the page aren't wondering who "our team" is.